### PR TITLE
Docs Updates

### PR DESCRIPTION
**Pull request by mjpierfinance**
- check_offers
- disbursements
- payment

All updated except for background status

### CODE REVIEW

File: fern/definition/coverage.yml [hunk 1]
```diff
@@ -271,25 +281,33 @@ types:
...
+      type: CheckOfferType
       amount:
         type: integer
         docs: The amount (in cents) of the offer (e.g., 300000 for $3,000.00).
       interest_rate:
         type: integer
         docs: The interest rate (in basis points) charged on the offer (e.g., 750 for 7.50%).
       term:
-        type: integer
+        type: optional<integer>
```
Comment (canadiaLite) [inline fern/definition/coverage.yml:292]: why is this marked optional?

File: fern/definition/coverage.yml [hunk 2]
```diff
@@ -271,25 +281,33 @@ types:
...
         docs: The term of the offer in months (e.g., 24 for a 24-month term).
       origination_fee:
         type: integer
         docs: The origination fee charged to originate the offer in basis points (e.g., 150 for 1.50%).
       first_payment_date:
-        type: string
+        type: optional<string>
```
Comment (canadiaLite) [inline fern/definition/coverage.yml:298]: why is this marked optional?

**Review by canadiaLite — APPROVED** [review]
lg. just double check if those attrs can actually be made optional or not

File: fern/definition/coverage.yml (hunk 2 above)
Comment (mjpierfinance) [reply fern/definition/coverage.yml:298]: LOC don't have these

File: fern/definition/coverage.yml (hunk 1 above)
Comment (mjpierfinance) [reply fern/definition/coverage.yml:292]: LOC don't have these

**Pull request merged**